dev/Pager/PrototypePager/Themes/PrototypePager.xaml

```diff
@@ -0,0 +1,181 @@
+                                    <VisualState.Setters>
+                                        <Setter Target="NumberBoxDisplay.IsEnabled" Value="True"/>
+                                        <Setter Target="BoxPanels.Visibility" Value="Visible"/>
+                                        <Setter Target="NumberBoxDisplay.Visibility" Value="Visible"/>
+                                    </VisualState.Setters>
+                                </VisualState>
+                                <VisualState x:Name="ComboBoxVisible">
+                                    <VisualState.Setters>
+                                        <Setter Target="ComboBoxDisplay.IsEnabled" Value="True"/>
+                                        <Setter Target="BoxPanels.Visibility" Value="Visible"/>
+                                        <Setter Target="ComboBoxDisplay.Visibility" Value="Visible"/>
+                                    </VisualState.Setters>
+                                </VisualState>
+                                <VisualState x:Name="NumberPanelVisible">
+                                    <VisualState.Setters>
+                                        <Setter Target="BoxPanels.Visibility" Value="Collapsed"/>
+                                        <Setter Target="NumberBoxDisplay.Visibility" Value="Collapsed"/>
+                                        <Setter Target="ComboBoxDisplay.Visibility" Value="Collapsed"/>
+                                        <Setter Target="NumberPanelDisplay.Visibility" Value="Visible"/>
+                                    </VisualState.Setters>
+                                </VisualState>
+                            </VisualStateGroup>
+                        </VisualStateManager.VisualStateGroups>
+                        <!--Spacing="{TemplateBinding Spacing}"-->
+                            <StackPanel x:Name="BackButtons"
+                                        Orientation="Horizontal"
+                                        Grid.Row="0"
+                                        Grid.Column="0"
+                                        HorizontalAlignment="Right">
+                                <Button x:Name="FirstPageButton"
+                                        IsEnabled="False"
+                                        Style="{StaticResource NavigationBackButtonNormalStyle}"
+                                        FontFamily="Segoe MDL2 Assets"
+                                        Content="&#xE892;"/>
+                                <TextBlock x:Name="FirstPageButtonText" VerticalAlignment="Center" HorizontalAlignment="Center"
+                                           Text="{TemplateBinding FirstPageButtonText}"
+                                           Visibility="{Binding ElementName=FirstPageButton, Path=Visibility}"/>
+                                <Button x:Name="PreviousPageButton"
+                                        IsEnabled="False"
```

Comment:
>False [](start = 51, length = 5)

disabled by default?

Reply:
I disabled the first and previous page buttons because when the pager is loaded in, it will begin on page 1. You can't really use these buttons when you're on the first page so I disabled them in the xaml file.

Would you suggest not messing with the IsEnabled property in the template and leaving the button enabled and disabling it from codebehind when it is loaded in?

Reply:
I think I would suggest setting the property via VSM. We should almost never update properties in code behind.

---
In reply to: [463892303](https://github.com/microsoft/microsoft-ui-xaml/pull/2977#discussion_r463892303) [](ancestors = 463892303)

Reply:
Also, it is possible for the pager to load on a different page right? if you have <Pager SelectedIndex="4"/>

---
In reply to: [465345248](https://github.com/microsoft/microsoft-ui-xaml/pull/2977#discussion_r465345248) [](ancestors = 465345248,463892303)

Reply:
Yes, you can load on a different page by setting the SelectedIndex property of the pager.

Reply:
Added in Visual States for the IsEnabled properties!